Assets/RedBlueGames/MulliganRenamer/Editor/BulkRenamer.cs

```diff
@@ -268,8 +268,8 @@ private static void CacheAsestsInSameDirectoryAsAsset(UnityEngine.Object asset,
 
         private static void UpdateProgressBar(int currentStep, int totalNumSteps)
         {
-            var infoString = string.Format("Renaming Object {0} of {1}", currentStep++, totalNumSteps);
-            EditorUtility.DisplayProgressBar("Renaming...", infoString, currentStep / (float)totalNumSteps);
+            var infoString = string.Format(LocaleManager.Instance.GetTranslation("renamingObjectXofY"), currentStep++, totalNumSteps);
+            EditorUtility.DisplayProgressBar(LocaleManager.Instance.GetTranslation("renaming")+ "...", infoString, currentStep / (float)totalNumSteps);
```

Comment:
I trust your judgement on separating out the "..." - did you do that just because it's not really part of the statement, and just meant to mean "this will take some time"?

Reply:
Since the ellipsis is known worldwide (I might be wrong about this), I decided to not localize this symbol and just localize the word "renaming". 
I also think this is better because we can reuse the translation in other places without creating a new localization key to have the word "renaming" without the ellipsis.

Reply:
Sounds good. Was just curious. I reviewed our localization on some strings in Sparklite, and they seem to confirm you're right that it's universal.